module/os/linux/spl/spl-taskq.c

```diff
@@ -1031,6 +1039,7 @@ taskq_create(const char *name, int nthreads, pri_t pri,
 	taskq_thread_t *tqt;
 	int count = 0, rc = 0, i;
 	unsigned long irqflags;
+	int proper_nthreads = nthreads;
```

Comment:
It might be more clear if we rename the `nthreads` arg to something like `threads_arg` and then have a local variable `nthreads` which retains its old meaning of "how many threads we're creating now".